modcc/io/prefixbuf.cpp

```diff
@@ -75,10 +75,14 @@ int indent_manip::xindex() {
 
 static void apply_indent_prefix(std::ios& s, int index) {
     if (auto pbuf = dynamic_cast<prefixbuf*>(s.rdbuf())) {
-        indent_stack* stack_ptr = static_cast<indent_stack*>(s.pword(index));
-        size_t tabs = (!stack_ptr || stack_ptr->empty())? 0 : stack_ptr->top();
-        size_t tabwidth = s.iword(index);
-        pbuf->prefix = std::string(tabs*tabwidth, ' ');
+        if (auto ptr = static_cast<indent_stack*>(s.pword(index));
+            ptr && !ptr->empty()) {
+            unsigned n_tab = ptr->top();
```

Comment:
Argh! This is terribly misleading formatting

Reply:
I knew you were going to love it. Isn't it beautiful how the second line of the condition and the first line of the condition line up?

Reply:
It has a certain horrible charm to it.

Reply:
Fixed it.